draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -22,16 +22,31 @@ export default class Toolbar extends React.Component {
 
   state = {
     isVisible: false,
+    position: undefined,
+
+    /**
+     * If this is set, the toolbar will render this instead of the regular
+     * structure and will also be shown when the editor loses focus. It's
+     * the responsibility of the callee to call this function again with
+     * `undefined` to show the regular structure again or hide the toolbar
+     * if the editor doesn't have focus anymore.
+     *
+     * @type {Component}
+     */
+    overrideContent: undefined
```

Comment:
I was thinking if this should be a `Component` or an `Element`, but since the `structure` also consists of `Component`s i went with that to be consistent.